melosys-eessi-app/src/main/java/no/nav/melosys/eessi/kafka/producers/model/MelosysEessiMelding.java

```diff
@@ -23,6 +24,7 @@
     private String artikkel;
     private boolean erEndring;
     private boolean midlertidigBestemmelse;
+    private boolean erMottaksInstitusjon;
```

Comment:
For å skille på at dette med X006 er litt special-case (vil skje sjeldent, og kun for én SED-type) og for å skape mindre forvirring, kanskje vi skal prefikse navnet slik at vi får noe ala `x006NavErFjernet`? 👀 🤷

Reply:
Skjønner, hva med feks erX006Mottaker?

Reply:
Tror det hadde hjulpet på lesbarhet av kode å faktisk få konteksten for hva som skjer når NAV er mottaker - at vi da altså er fjernet fra BUCen og ikke skal registrere en eventuell periode tilhørende BUCen.

Å kun spesifisere at man er mottaker tror jeg også blir litt feil i og med vi fant ut at vi uansett mottar SEDen. Vi er jo faktisk interessert i om vi er fjernet fra BUCen eller ikke.

Reply:
Ok